package.json

```diff
@@ -47,7 +47,7 @@
       }
     ],
     "javaExtensions": [
-      "./server/com.microsoft.java.debug.plugin-0.29.0.jar"
+      "./server/com.microsoft.java.debug.plugin-0.30.0.jar"
```

Comment:
Is `30` here on purpose?

Reply:
yes. In latest debugger release, we didn't modify the code in java-debug. So there is one version gap between them.